### PR TITLE
dev: Updated packages organization

### PR DESCRIPTION
**Pull request by akhercha**
Closes: #NA

## Added
- `pragma-consumer` placeholder new crate

## Updated
- Moved every dependency versions to the main `Cargo.toml` (so we never have versions mismatch + easier to update)
- Updated rust to `1.80`
- Updated Dockerfiles

### CODE REVIEW

**Comment by akhercha** [comment]
@hel-kame Can you quickly check the new Dockerfiles, see if everything's correct? They work locally. 🙏

File: Cargo.toml [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+tokio = { version = "1.11.0", features = ["full"] }
+toml = "0.8.8"
+utoipa = "4"
+utoipa-swagger-ui = { version = "4", features = ["axum"] }
+uuid = { version = "1.4", features = ["fast-rng", "v4", "serde"] }
+
+pragma-monitoring = { git = "https://github.com/astraly-labs/pragma-monitoring", branch = "main" }
+
+# Test dependencies
```
Comment (EvolveArt) [inline Cargo.toml:58]: add it in dev dependencies

**Comment by hel-kame** [comment]
LGTM 😎

**Review by hel-kame — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (akhercha) [reply Cargo.toml:58]: not possible unfortunately - checked deoxys & madara orchestrator & they have the same problem, it needs to be in `[workspace.dependencies]`  and the sub crate imports like as `[dev-dependencies]`

**Pull request merged**